mmv1/third_party/terraform/tests/resource_compute_region_network_endpoint_group_test.go.erb

```diff
@@ -0,0 +1,78 @@
+<% autogen_exception -%>
+package google
+
+<% unless version == 'ga' -%>
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+<% end -%>
+
+<% unless version == 'ga' -%>
+func TestAccComputeRegionNetworkEndpointGroup_negWithServerlessDeployment(t *testing.T) {
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"random_suffix": randString(t, 10),
+	}
+
+	vcrTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckComputeRegionNetworkEndpointGroupDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccComputeRegionNetworkEndpointGroup_negWithServerlessDeployment(context),
+			},
+			{
+				ResourceName:            "google_compute_region_network_endpoint_group.test_neg",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"region"},
+			},
+		},
+	})
+}
+<% end -%>
+
+<% unless version == 'ga' -%>
+func testAccComputeRegionNetworkEndpointGroup_negWithServerlessDeployment(context map[string]interface{}) string {
+	return Nprintf(`
+resource "google_api_gateway_api" "api_gw" {
+	api_id = "api-gw"
```

Comment:
Can we change all the user set ID and names in this test config to the format:

`tf-test-%{random_suffix}`? It allows our CI that runs tests to delete old resources that are left around, and not run into issues with conflicting names if this test is run twice at the same time

Reply:
Fixed